src/schedule/schedule_lang.cc

```diff
@@ -397,6 +397,25 @@ Stage& Stage::double_buffer() {
   return *this;
 }
 
+Stage& Stage::opengl() {
+  CHECK(!is_scheduled()) << "Must be a fresh schedule";
+  StageNode *self = operator->();
+
+  auto all_iter_vars = self->all_iter_vars; // curr version of all_iter_vars
```

Comment:
be careful that this can include reduction variables, which should not be part of the fusing stage

Reply:
Will go back to this later. Added a TODO for now.